### PR TITLE
Section styles for mobile

### LINKED ISSUE #265: Style: Scaling for sections styles desktop → mobile
In mobile view, the font-sizes and line heights are shrinking. The default style system should take desktop to mobile into consideration. 

The following rules were discovered so far, needs to be double-checked and enhanced if applicable and implemented:

* [x] h2, p : 28.8 → 20.8 (line-height 31.2)
* [x] h1 40 → 32

text with logosecondary line-height: 43.2 → 31.2px

### PR DESCRIPTION
**Pull request by Acidham**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #265 

Test URLs:
- Before: 
- https://main--mammotome--hlxsites.hlx.page/us/en/healthcare-professionals/webinars
- https://main--mammotome--hlxsites.hlx.page/us/en/products/mammotome-confirm

- After: 
- https://265-style-scaling-for-mobile--mammotome--hlxsites.hlx.page/us/en/healthcare-professionals/webinars
- https://265-style-scaling-for-mobile--mammotome--hlxsites.hlx.page/us/en/products/mammotome-confirm

### CODE REVIEW

File: styles/styles.css [hunk 1]
```diff
@@ -661,10 +680,18 @@ main .section[data-section-status='initialized'] {
...
-  font-size: var(--mt-heading-font-size-xxl);
+  font-size: var(--mt-heading-font-size-xl);
   line-height: 2.5rem;
 }
 
+@media screen and (min-width: 767px) {
+  .section.callout h1,
+  .section.callout h2,
+  .section.callout h3 {
+    font-size: var(--mt-heading-font-size-xxl);
```
Comment (dzehnder) [inline styles/styles.css:691]: Same font size for h1-h3?

**Review by dzehnder — DISMISSED** [review]

**Review by chicharr — CHANGES_REQUESTED** [review]
@Acidham changes look good to me. However, there are some differences in the appearance of the headings compared to Mammotome.com. Please check https://github.com/hlxsites/mammotome/pull/279#issuecomment-1597223058

File: styles/styles.css (hunk 1 above)
Comment (Acidham) [reply styles/styles.css:691]: this is how @smiroslav  implemented it

**Comment by Acidham** [comment]
@chicharr yes it looks different, but MT uses different sizes for desktop and mobile. When I change a downsizing of the font for H1 for example, it must be the same across the page. Webinars behaves now same like Mammotome Confirm.

**Review by chicharr — APPROVED** [review]
@Acidham As agreed on slack, please open a new issue to tackle the padding in the headings for the product pages on mobile. See https://main--mammotome--hlxsites.hlx.page/us/en/products/mammotome-confirm.
In original site, in mobile view, the heading 1 is split in 2 lines due to the padding

**Review by chicharr — APPROVED** [review]
LGTM. As discussed on slack this change is setting a uniform h1 font-size across products and other pages. This is in practice making the title on the Webinars page smaller.
@Acidham  as please ask Krizel if she is ok to make the heading 1 font-size uniform for products and other pages

**Pull request merged**